src/reconstruction/reconstruct-2d-ftu.js

```diff
@@ -0,0 +1,84 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { info, error } from '../utils/logging.js';
+import { writeReconstructedData, executeBlazegraphQuery, loadGraph, shortenId, quoteIfNeeded } from './utils.js';
+
+export function reconstruct2dFtu(context) {
+  try {   
+    loadGraph(context);
+    queryGraph(context);
+    transformRecords(context);
+
+    info('2D FTU reconstruction completed successfully');
+  } catch (err) {
+    error('Error during 2D FTU reconstruction:', err);
+    throw err;
+  }
+}
+
+function queryGraph(context) {
+  try {
+    const processorHome = resolve(context.processorHome);
+    const reconstructPath = resolve(context.reconstructionHome);
```

Comment:
Using the same directory and clearing it all the time will cause major issues if we want to do things in parallel. Please refactor so that each DO works in its own directory. Why can't this happen in the 'reconstructed' directory?